src/locales/__tests__/i18n-formatting.test.ts

```diff
@@ -19,7 +19,7 @@ test('formatting dates with i18n', () => {
           timeZoneName: 'short',
           timeZone: 'UTC',
           locale: 'en-US',
-        },
+        } satisfies Intl.DateTimeFormatOptions & { locale?: Intl.BCP47LanguageTag },
```

Comment:
Shouldn't we have a name for this union?

Reply:
I think that's needlessly complex just for tests. Unfortunately I'm not able to make a strict type with 
```ts
      (Intl.DateTimeFormatOptions & { locale?: Intl.BCP47LanguageTag })
    | (Intl.RelativeTimeFormatOptions & { locale?: Intl.BCP47LanguageTag })
    | (Intl.NumberFormatOptions & { locale?: Intl.BCP47LanguageTag })
    | (Intl.ListFormatOptions & { locale?: Intl.BCP47LanguageTag })
```

Reply:
or i could just remove locale  https://github.com/oasisprotocol/explorer/pull/115/commits/3f951c4d38e17d87e3c1b1b8f64b73613256cc99